dev_docs/tutorials/generating_oas_for_http_apis.mdx

```diff
@@ -155,6 +155,124 @@ function registerFooRoute(router: IRouter, docLinks: DoclinksStart) {
 }
 ```
 
+##### Adding examples
+
+Beyond the schema of requests and responses, it is **very useful** to provide
+concrete requests and responses as examples. Examples go beyond defaults and
+provide a more intuitive understanding for end users in learning the behaviour
+of your API. See the [bump.sh documentation](https://docs.bump.sh/guides/openapi/specification/v3.1/data-models/examples/)
+for more information on how examples will be shown to end users.
+
+To add examples to the endpoint we created above you could do the following:
```

Comment:
Recommending the code-first approach to keep consistent with the rest of the platform provided approach to OAS.